contrib/format-hdf5/src/main/java/org/apache/drill/exec/store/hdf5/HDF5BatchReader.java

```diff
@@ -410,8 +390,22 @@ public boolean next() {
     return true;
   }
 
+  @Override
+  public void close() {
+    AutoCloseables.closeSilently(hdfFile, reader);
+    /*
+     * The current implementation of the HDF5 reader creates a temp file which needs to be removed
+     * when the batch reader is closed. A possible future functionality might be to used to.
```

Comment:
seems an incomplete comment: "used to" do what?

Reply:
Done, I added new text for the note.